src/rendering/screenspacerenderable.cpp

```diff
@@ -70,12 +70,11 @@ namespace {
     {
         "UseRadiusAzimuthElevation",
         "Use Radius Azimuth and Elevation",
-        "This value determines whether the location of this screen space plane will be "
-        "specified using radius, azimuth and elevation (if this is set to 'true') or "
-        "using cartesian coordinates. By switching this value, the correct property will "
-        "be shown or hidden. The Cartesian coordinate system is useful if a regular "
-        "rendering is applied, whereas the radius azimuth elevation are most useful in a "
-        "planetarium environment.",
+        "Determines whether the location of this screen space plane will be specified "
```

Comment:
Do we want to say screen space or screenspace? Above there is "screenspace"

Reply:
The renderable is called "ScreenSpace" with two words, so my vote goes to "screen space". 

But I guess we should discuss

Reply:
Nvm. I'll just update it to be "screen space" consistently. It is and should be two words